transport/anonymizer.go

```diff
@@ -36,15 +36,15 @@ func NewAnonymizer[T any](
 	}
 }
 
-func (a *Anonymizer[T]) New(name stream.EventName) (any, error) {
+func (a *Anonymizer[T]) New(name string) (any, error) {
```

Comment:
does this mean that now `name` is not the name of the event that `Anonymizer` can handler, but the name of the `Anonymizer`? That's what the code suggests I think

Reply:
According to the mapper interface it should return a new instance of an event corresponding to the event name: https://github.com/ThreeDotsLabs/esja/blob/stream/transport/mapper.go

Reply:
I think we could name the argument eventName now for clarity?

Reply:
Fixed